x-pack/plugins/observability_solution/infra/server/lib/alerting/inventory_metric_threshold/lib/is_rate.ts

```diff
@@ -37,7 +37,7 @@ export const isInterfaceRateAgg = (metric: MetricsUIAggregation | undefined) =>
   const values = Object.values(metric);
   return (
     values.some((agg) => ESTermsWithAggregationRT.is(agg)) &&
```

Comment:
Forgot about this one. Let's try to replace this too.

Reply:
I had a chat with Jenny earlier today, and we weren't sure about replacing `ESTermsWithAggregationRT`. I'll do it then. Thanks